### PR TITLE
Draft: Add GPG namespace

### PR DESCRIPTION
**Pull request by sebastinez**
This PR proposes the implementation of the GPG namespace.

Discussion is located in #65

### CODE REVIEW

File: CAIPs/caip-draft-gpg.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+---
+caip: X
+title: CAIP-10 for the GPG Namespace
+author: Sebastian Martinez (@sebastinez)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/issues/65, https://github.com/ChainAgnostic/CAIPs/pull/53
+status: Draft
+type: Standard
+created: 2021-09-02
+updated: 2021-09-02
+requires: X
```
Comment (ligi) [inline CAIPs/caip-draft-gpg.md:10]:
```suggestion
requires: 2
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by bumblefudge** [comment]
Discussed at last meeting, leaving open pending as-yet undefined Namespaces policy on non-chain crypto systems. At this meeting, we also noted we should document better where new namespace definiions will go (/namespaces/).  We also discussed the distinction between caip:gpg and caip:gpg-vks (https://keys.openpgp.org/about/api)-- how VKS could fit was discussed.

**Comment by bumblefudge** [comment]
Removed next meeting tag and will return to this issue after editorial namespace overhaul and discussions of non-chain boundary of this group's scope

**Comment by silverpill** [comment]
What's the status of this?
Are non-blockchain namespaces allowed?

**Comment by obstropolos** [comment]
Doing some cleanup here - we've now moved all namespace requests to the namespaces repo. If this discussion is to continue, please reopen this in the namespaces repo: https://github.com/chainagnostic/namespaces - thanks!

**Pull request closed without merging**